### ISSUE TITLE
מידע על כספי ממשלה של ארגון

### ISSUE THREAD

**Issue by maryLoi** [OpenBudget/BudgetKey#504]
החברה ביצעה מכרזים אבל למעלה לא כתוב שקיבלה כספי ממשלה

https://next.obudget.org/i/org/company/513469460?li=0

![image](https://user-images.githubusercontent.com/8996235/95251570-6cac2d00-0824-11eb-840e-0400fe02f336.png)

**Comment by akariv** [comment]
למיטב זכרוני המספר הזה הוא רק עבור 3 השנים האחרונות

**Comment by maryLoi** [comment]
@akariv 
הסיפור הזה של 3 שנים הוא מאוד מבלבל ולא אינטואיטיבי.
הייתי מציעה לעשות סה"כ בכל השנים , ואם לא אז רק בשנה האחרונה

**Comment by noamoss** [comment]
@akariv 

>למיטב זכרוני המספר הזה הוא רק עבור 3 השנים האחרונות

הייתה לזה איזו סיבה מקצועית... לא?

**Comment by akariv** [comment]
כן, כי להציג את הסה״כ כשלא ברור ממתי הוא (ועבור כל גוף ופריט זה משתנה) זה
מבלבל לא פחות.
המטרה היתה לאפשר השוואת ״תפוחים לתפוחים״ ולא להציג מידע שאינו בר השוואה וגם
היסטורית פחות רלוונטי.
